js/mobile-menu.js

```diff
@@ -1,46 +1,223 @@
-    // Show desktop footer and hide contact form
-    desktopFooter.style.display = 'block';
-    contactForm.style.display = 'none';
-
-    // Show desktop version and hide mobile version
-    desktopVersion.style.display = 'block';
-    mobileVersion.style.display = 'none';
-  } else {
-    // Show contact form and hide desktop footer
-    desktopFooter.style.display = 'none';
-    contactForm.style.display = 'block';
-
-    // Show mobile version and hide desktop version
-    desktopVersion.style.display = 'none';
-    mobileVersion.style.display = 'block';
+const projectButton = document.querySelectorAll('.project-button');
+const projectButtoni = document.querySelectorAll('.project-buttoni');
+const menuButton = document.getElementById('menu-button');
+const popupSection = document.querySelector('#popups');
+const aboutSection = document.querySelector('#about2');
+const aboutiSection = document.querySelector('#about');
+
+const abouts = [
+  {
+    id: 1,
+    title: 'Multi - Post stories',
+    description: `A daily selection of privately personalized reads; no accounts or sign-ups required.
+    has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: 2,
+    title: 'Multi - Post stories',
+    description: `A daily selection of privately personalized reads; no accounts or sign-ups required.
+    has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: 3,
+    title: 'Multi - Post stories',
+    description: `A daily selection of privately personalized reads; no accounts or sign-ups required.
+    has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+  {
+    id: 4,
+    title: 'Multi - Post stories',
+    description: `A daily selection of privately personalized reads; no accounts or sign-ups required.
+    has been the industry's standard dummy text ever since the 1500s, when an unknown
+    printer took a standard dummy text.`,
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
+];
+
+const aboutHtmlTemplate = (about) => {
+  return `
+            <div>
+              <div class="wideimg"></div>
+              <h2 class="multi-post">${about.title}</h2>
+              <p class="story">
+              ${about.description}
+              </p>
+              <ul class="list">
+              ${about.technologies
+                .map((tech) => `<li class="lang">${tech}</li>`)
+                .join(`<li class="vertical"></li>`)}
+              </ul>
+              <button class="button project-button" id="project-button1">
+                See Project
+              </button>
+            </div>
```

Comment:
Let's try to format this better for more readability

Reply:
#### This is a very sensitive notification, it has been resolved.